tests/test_benchmark_simple_scenarios.py

```diff
@@ -118,6 +118,47 @@ def test_benchmark_ABE_grid_pv_bat(self, margs):
         # compare the total excess electricity between the two cases
         assert excess["AB_grid_PV"] < excess["ABE_grid_PV_battery"]
 
+        @pytest.mark.skipif(
+            EXECUTE_TESTS_ON not in (TESTS_ON_MASTER),
+            reason="Benchmark test deactivated, set env variable "
+            "EXECUTE_TESTS_ON to 'master' to run this test",
+        )
+        @mock.patch(
+            "argparse.ArgumentParser.parse_args", return_value=argparse.Namespace()
+        )
+        def test_benchmark_AD_grid_diesel(self, margs):
+            # define the two cases needed for comparison (grid + PV) and (grid + PV + battery)
+            use_case = "AD_grid_diesel"
+            # define an empty dictionary for excess electricity
+            main(
+                overwrite=True,
+                display_output="warning",
+                path_input_folder=os.path.join(TEST_INPUT_PATH, use_case),
+                input_type=CSV_EXT,
+                path_output_folder=os.path.join(TEST_OUTPUT_PATH, use_case),
+            )
+            # read timeseries_all_busses excel file
+            busses_flow = pd.read_excel(
```

Comment:
Did you always use the excel to verify your benchmark tests? We will have to change that sometime in the future, so that the MVS can be tested based on the json output.

Reply:
yes at the moment this method is used for all of them